wandb/wandb_torch.py

```diff
@@ -74,8 +74,9 @@ def parameter_log_hook(module, input_, output):
         # This won't handle the case if the network changes
         if log_gradients:
             for name, parameter in module.named_parameters():
-                self._hook_variable_gradient_stats(
-                    parameter, 'gradients/' + prefix + name)
+                if parameter.requires_grad:
```

Comment:
I'm not sure "requires_grad" exists in every PyTorch version. Maybe not in 0.3

Reply:
ah good catch

Reply:
actually the pytorch 0.3 case works because Tensors are replaced by Variables everywhere